WebApp/app/domain/App/blockchainUtils.ts

```diff
@@ -68,7 +68,7 @@ export const getNetworkId = async (): Promise<number> => {
   try {
     return await getWeb3().eth.net.getId();
   }catch(error) {
-    return 99
+    return 90909
```

Comment:
What is this magic number?

Reply:
The chain Id, i changed it a different one because 99 is the chain id of Poa Network (according to https://chainid.network/chains/) , but now that you bring this to attention, I'm thinking that instead of dealing with magic numbers, to leave the exception to bubble up or something if the network id is not available. Will do a test to see what happens.